Standard.AI.Data.EntityIntelligence/Brokers/Datas/DbContext.cs

```diff
@@ -0,0 +1,29 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+using Dapper;
+using Standard.AI.Data.EntityIntelligence.Models.Datas;
+
+namespace Standard.AI.Data.EntityIntelligence.Brokers.Datas
+{
+    internal class DbContext
```

Comment:
`DbContext` if not a library it doesn't have a place in a Standard-compliant system. It needs to fall under one of these categories in behavior:
- Brokers
- Services
- Exposers

or simply be a data-carrier model.